atoms/set-5/client/js/app.js

```diff
@@ -110,4 +110,35 @@ function initCardSections() {
   });
 }
 
+function invertColorOnIframeResultPages(event) {
```

Comment:
There don't seem to have been any changes to set-5 main.html, so is this code change necessary?  Or has it been added for consistency & future-proofing with the other sets?